README.md

```diff
@@ -18,64 +17,75 @@ Then, add one or more of the modules below:
-| Akka support | out of the box | requires custom serializer | requires custom serializer | requires custom serializer | out of the box |
+| Akka support | out of the box | requires custom serializer | requires custom serializer | requires custom serializer | requires custom serializer |
 | Runtime safety | none <ul><li>uses reflection</li><li>errors appear only in runtime</li></ul> | encoders and decoders are checked during compile time | generates scala classes | standard for scala code | encoders and decoders are checked during compile time
-| Schema evolution | <ul><li>removing field</li><li>adding optional field</li></ul> with [`JacksonMigration`](https://doc.akka.io/docs/akka/current/serialization-jackson.html#schema-evolution) <ul><li>adding mandatory field</li><li>renaming field</li><li>renaming class</li><li>support of forward versioning for rolling updates</li></ul>| <ul><li>adding optional field</li><li>removing optional field</li><li>adding required field with default value</li><li>removing required field</li><li>renaming class</li></ul> | <ul><li>switching between optional and repeated field</li><li>adding new fields</li><li>renaming fields</li></ul> | <ul><li>reordering fields</li><li>renaming fields</li><li>adding optional field</li><li>adding required field with default value</li><li>removing field with default value</li></ul> | any arbitrary transformation can be defined manualy with transcoders
-| Boilerplate | a lot: <ul><li>ADTs requires amount of annotation equal to or exceeding the actual type definitions</li><li>poor support for case object - needs sealed trait per case object and a custom deserializer</ul> | none, Circe can derive codecs from case class definitions | in case of custom types a second layer of models is needed | sometimes requires annotations | considerable: every top level sealed trait must have manually defined codec
+| Schema evolution | <ul><li>removing field</li><li>adding optional field</li></ul> with [`JacksonMigration`](https://doc.akka.io/docs/akka/current/serialization-jackson.html#schema-evolution) <ul><li>adding mandatory field</li><li>renaming field</li><li>renaming class</li><li>support of forward versioning for rolling updates</li></ul>| <ul><li>adding optional field</li><li>removing optional field</li><li>adding required field with default value</li><li>removing required field</li><li>renaming field</li><li>reordering fields</li></ul> | <ul><li>switching between optional and repeated field</li><li>adding new fields</li><li>renaming fields</li></ul> | <ul><li>reordering fields</li><li>renaming fields</li><li>adding optional field</li><li>adding required field with default value</li><li>removing field with default value</li></ul> | only renaming fields
```

Comment:
Hmm for Borer: _only renaming fields_ refers to what's provided out of the box... but all other operations are possible as well, right?

Reply:
I suppose that's the case for all other libs: that the row lists what is provided OOTB rather than what's possible in principle (?)... feel free to contact the original author(s) of this table if needed